### ISSUE TITLE
Improve install documentation

### ISSUE THREAD

**Issue by mikalstill** [shakenfist/shakenfist#1171]

**Comment by dth99** [comment]
yes, CentOs or RHEL are beyond good. If we talk about components and speed/performance there is no competition of RedHat based Operating Systems in Market.

**Comment by mikalstill** [comment]
Closing during a triage sweep of old issues: every specific complaint here predates the current installer — deployment was rewritten as an Ansible collection and the docs site restructured since 2021. Install documentation can doubtless still improve, but that needs fresh, specific issues against the current docs rather than this historical one.

*(Triage assisted by Claude Code; closure approved by Michael Still.)*
